packages/peregrine/lib/List/list.js

```diff
@@ -13,35 +13,43 @@ const List = props => {
     const {
         classes,
         getItemKey,
+        initialSelection,
         items,
         render,
         renderItem,
         onSelectionChange,
         selectionModel,
         ...restProps
     } = props;
+
     const customProps = {
         classes,
         getItemKey,
         items,
         onSelectionChange,
         selectionModel
     };
-    const Root = useMemo(
-        () => fromRenderProp(render, Object.keys(customProps)),
-        [render, customProps]
-    );
+
     const handleSelectionChange = useCallback(
         selection => {
-            onSelectionChange && onSelectionChange(selection);
+            if (onSelectionChange) {
+                onSelectionChange(selection);
+            }
```

Comment:
Yeah, definitely, both work.

Regarding readability in general I err on the side of clarity, which sometimes means having to be a little more verbose. I'd rather read something a little longer once and fully understand it than have to read something shorter multiple times and have to parse it.

That said, this isn't a great example since the previous method is common enough that I think most readers would probably understand it on first read.

Reply:
Raises hand, I wouldn't have understood the && outcome without Googling. Never seen that syntax before, and if I saw it again I'd have to think harder than an explicit if(). Two cents deposited.

Reply:
`&&` or `||` ([Short Circuiting](https://en.wikipedia.org/wiki/Short-circuit_evaluation)) is a very prominent coding ideology in many programming languages. But if the team is comfortable with `if()` let's do that.